modules/lang-painless/src/main/resources/org/elasticsearch/painless/java.util.regex.txt

```diff
@@ -29,5 +29,31 @@ class Pattern -> java.util.regex.Pattern extends Object {
 }
 
 class Matcher -> java.util.regex.Matcher extends Object {
+  int end()
+  int end(int)
+  boolean find()
+  boolean find(int)
+  String group()
+  String group(int)
+  # String group(String)  How do I get this working?
```

Comment:
we don't allow method signature overriding in this way, intentionally. Otherwise things would be much slower and more ambiguous...